contracts/0.8.25/vaults/OperatorGrid.sol

```diff
@@ -481,6 +482,81 @@ contract OperatorGrid is AccessControlEnumerableUpgradeable, Confirmable2Address
         return true;
     }
 
+    /// @notice Syncs vault tier with current tier params
+    /// @param _vault address of the vault
+    /// @return bool Whether the sync was confirmed.
+    /// @dev Requires vault to be connected to VaultHub to finalize tier sync.
+    /// @dev Both vault owner and node operator confirmations are required.
+    function syncTier(address _vault) external returns (bool) {
+        (VaultHub vaultHub, VaultHub.VaultConnection memory vaultConnection, address vaultOwner, address nodeOperator) = _getVaultContextForConnectedVault(_vault);
+
+        ERC7201Storage storage $ = _getStorage();
+        uint256 vaultTierId = $.vaultTier[_vault];
+        Tier storage tier = $.tiers[vaultTierId];
+
+        if (
+            vaultConnection.reserveRatioBP == tier.reserveRatioBP &&
+            vaultConnection.forcedRebalanceThresholdBP == tier.forcedRebalanceThresholdBP &&
+            vaultConnection.infraFeeBP == tier.infraFeeBP &&
+            vaultConnection.liquidityFeeBP == tier.liquidityFeeBP &&
+            vaultConnection.reservationFeeBP == tier.reservationFeeBP
+        ) {
+            revert VaultAlreadySyncedWithTier();
+        }
+
+        // store the caller's confirmation; only proceed if the required number of confirmations is met.
+        if (!_collectAndCheckConfirmations(msg.data, vaultOwner, nodeOperator)) return false;
+
+        vaultHub.updateConnection(
+            _vault,
+            vaultConnection.shareLimit,
+            tier.reserveRatioBP,
+            tier.forcedRebalanceThresholdBP,
+            tier.infraFeeBP,
+            tier.liquidityFeeBP,
+            tier.reservationFeeBP
+        );
+
+        emit TierSynced(_vault);
```

Comment:
🤔 maybe not needed, because storage of OperatorGrid hasn't changed and we have event in `updateConnection`